### ISSUE TITLE
Project Update

### ISSUE THREAD

**Issue by Rigellute** [Rigellute/spotify-tui#1000]
I sincerely apologise for my lack of maintenance, I do love this project but I have not been able to dedicate any time to it - life/other commitments have been getting in the way! I hope to be able to make more space for development in the future.

A serious amount of work is needed to upgrade the underlying spotify library. What's more, the current architecture has grown in complexity and needs a rethink. I had spent time exploring this refactor, but ran out of steam trying to reimplement existing features. 

Just know that I do intend to return to it!

**Comment by Rigellute** [comment]
It's rather fitting that this is issue #1000 😁

**Comment by sewnie** [comment]
### **LETS FUCKING GOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO**

**Comment by elijahmg** [comment]
@Rigellute Thx for your work. I faced some issues because of underlying library and since I needed only couple of feature, I've started to build my own pure cli tool. It's very mature. 
Here it is https://github.com/elijahmg/spotify-rust-cli
Thank you for inspiration

**Comment by nopeless** [comment]
@Rigellute I feel like there are just too many bugs for this project to be viable for every day use and way many edge cases that cause discomfort

I really do like the concept of this project but for now, I feel like someone has to rewrite the codebase with better underlying ui engine

**Comment by Oppen** [comment]
Hi @Rigellute. Thanks for all the work you put on this project. I'd like to say I love it but I got too late to the party to try it :( I'm sure I would tho :heart:
I was thinking, is there anything we can do to help keep the project alive? I thought maybe we could create a GitHub org so some more people can review and merge PRs. I see a lot of people has enough interest to contribute fixes, so what's missing the most is a few people in charge of the follow up.
Of course, my suggestion only applies if you agree, and setting you as owner (you deserve it), just with some other people helping to keep it afloat.

**Comment by aromeronavia** [comment]
Yes, we are quite a few people interested in contributing to this <3

**Comment by sewnie** [comment]
Me waiting for @Rigellute to have someone takeover the project or have a organization 💀 

Honestly, can someone here responsible enough make a fork of this project that takes PRs and accepts contributions, and fixes to this project? Many parts of it is broken (1000+ issues relating to broken)

**Comment by sewnie** [comment]
i have now created a fork of this repository that lives at https://github.com/spotify-tui/spotify-tui; with some PRs merged and a cargo.lock update, but i do not know Rust if at all so i have applied some that do work.

**Comment by sewnie** [comment]
edit: https://github.com/aome510/spotify-player

**Comment by hoomp3** [comment]
> edit: https://github.com/aome510/spotify-player

this one is so good, definitely recommend everyone use this instead

**Comment by sewnie** [comment]
EVERYONE should use this anyway, the developer is super nice and extremely active, plus tui looks much better and has way more features that can actually replace the propietary spotify client.

**Comment by Rigellute** [comment]
Sorry again for my radio silence. I've been unable to spend any time on `spotify-tui` 😞. Consider the project in a deep sleep until I can return to make the needed, but considerable, updates/fixes

**Comment by ErtanTaner** [comment]
Thank you for all your work but if this project not going to be updated, can we make this as archive?

**Comment by yada97** [comment]
Hope you okay
